common/src/main/java/net/draycia/carbon/common/messages/CarbonMessages.java

```diff
@@ -180,6 +180,14 @@ void whisperTargetNotSet(
     @Message("nickname.set.others")
     void nicknameSetOthers(final Audience audience, final String target, final Component nickname);
 
+    @Message("nickname.error.character_limit")
+    void nicknameErrorCharacterLimit(
+        final Audience audience,
+        final Component nickname,
+        @Placeholder("min_limit") final int minLimit,
```

Comment:
I don't think you need to use `@Placeholder` here, the placeholder resolver should turn the names to `snake_case` automatically now.